### DISCUSSION TITLE
How many words are required for a new language?

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#2466] [Questions] [answered]
What is the minimum amount of words for a new language?

**Comment by unrealapex** [comment, marked as answer]
The language files(`language.json`) usually consist of 200 words. See the [`languages`](https://github.com/Miodec/monkeytype/tree/master/frontend/static/languages) directory.

**Reply by tajhans** [reply]
Thanks

**Reply by unrealapex** [reply]
No problem
